featuretools/tests/profiling/dfs_profile.py

```diff
@@ -0,0 +1,38 @@
+"""
+dfs_profile.py
+
+Helper module to allow profiling of the dfs operations.  At some point we may
+want to use pstats to output the results to a log, but I'm anticipating that
+LookingGlass will provide the performance data we want.
+
+Notes:
+  - output currently goes to the root directory and is in dfs_profile.stats
+  - *.stats is gitignored
+  - it uses the demo customers dataset for testing
+  - max_depth > 2 is very slow (currently)
+  - stats output can be viewed online with https://nejc.saje.info/pstats-viewer.html
+"""
+import cProfile
+from pathlib import Path
+
+import featuretools as ft
```

Comment:
Just out of curiosity: when is it better to use `import featuretools as ft ` v.s. `from featuretools import ... `? The tests in `featuretools/tests` mostly use `from ... import ...`. Is there any one in particular that should be favored going forward?

Reply:
tbh, I was just following the pattern in the example docs.  There's no particular benefit other than keeping things in separate namespaces